### PR TITLE
[#1142] added install pylint for jenkin.sh

### PR DESCRIPTION
**Pull request by sungpeo**
related to #1142

### CODE REVIEW

File: tools/jenkins/jenkins.sh [hunk 1]
```diff
@@ -45,6 +45,9 @@ build_sqoop
 
 make apps
 
+build/env/bin/pip install pylint
```
Comment (romainr) [inline tools/jenkins/jenkins.sh:48]: Thanks!

Just if there is already a pylint, shound't we skip this install?

Comment (sungpeo) [reply tools/jenkins/jenkins.sh:48]: Okay. I write an additional commit.

File: tools/jenkins/jenkins.sh [hunk 2]
```diff
@@ -45,8 +45,10 @@ build_sqoop
 
 make apps
 
-build/env/bin/pip install pylint
-ln -s $(which pylint) build/env/bin/pylint
+if ! [ -x "$(command -v pylint)" ]; then
+    build/env/bin/pip install pylint
+fi
+ln -sf $(which pylint) build/env/bin/pylint
```
Comment (romainr) [inline tools/jenkins/jenkins.sh:51]: Also add it in the `if`?

Comment (sungpeo) [reply tools/jenkins/jenkins.sh:51]: Creating a symbolic link should be outside of `if`.

`Cleaning repo.` would delete any change from the commit.
ref. https://github.com/sungpeo/hue/blob/9489b486dcbeabd2df554a87b2942892298a5171/tools/jenkins/jenkins.sh#L31-L39

It means that `build/env/bin/pylint` would be deleted. (not `SKIP_CLEAN`)

**Comment by romainr** [comment]
Thanks!

**Pull request merged**